lib/redmine_merge_request_links/patches/issue_patch.rb

```diff
@@ -0,0 +1,22 @@
+module RedmineMergeRequestLinks
+  module Patches
+    module IssuePatch
+      def self.included(base)
+        base.class_eval do
+          has_and_belongs_to_many :merge_requests
+        end
+      end
+      def merge_requests_list
+        if Rails::VERSION::MAJOR >= 5
+          ApplicationController.render(partial: 'merge_request_links/column', locals: { merge_requests: merge_requests })
+        else
+          ActionController::Base.new.render_to_string(partial: 'merge_request_links/column', locals: { merge_requests: merge_requests })
+        end
+      end
```

Comment:
Handling this view concern in the model here feels a bit unexpected. Other (closed source) plugins I've seen extend [QueriesHelper#column_value](https://github.com/redmine/redmine/blob/f9d6dc3031b8f5d44cc3cf10c1e5e47ace5448be/app/helpers/queries_helper.rb#L240) (via alias method chaining). This would also get rid of the Rails version case distinction.

Reply:
I wasn't happy with it but didn't know how to solve it as I have no previous experience with ruby / rails. Thanks for pointing me in the right direction.